app/models/user.rb

```diff
@@ -17,6 +17,19 @@ def authorized_to?(access_request)
     end
   end
 
+  def inventory_groups
+    rbac_permissions.each_with_object([]) do |permission, ids|
+      next unless Rbac.verify(permission.permission, Rbac::INVENTORY_HOSTS_READ)
+      # Empty array on 'resource_definitions' symbolizes a global access to the permitted resource.
+      # In such case, the method returns Rbac::ANY and skips parsing of attributeFilter.
+      return Rbac::ANY if permission.resource_definitions == []
+
+      permission.resource_definitions.each do |filter|
+        ids.append(*Rbac.definition_value(filter[:attributeFilter], group: :id))
```

Comment:
As we no longer need the juggling with pluralization, this can be simplified:
```suggestion
        ids.append(*Rbac.definition_value(filter[:attributeFilter], 'group.id'))
```